src/App.js

```diff
@@ -402,86 +408,104 @@ class App extends React.Component {
-                      walletProviderId={walletProviderId}
-                    />
-                  </div>
-
-                  <HelpScoutBeacon locator={locator} apps={apps} />
-                </ActivityProvider>
-              </FavoriteDaosProvider>
-            </LocalIdentityModalProvider>
-          </ModalProvider>
-        </IdentityProvider>
-      </HelpScoutProvider>
+                    >
+                      <LocalIdentityModal
+                        address={intentAddress}
+                        label={intentLabel}
+                        opened={identityIntent !== null}
+                        onCancel={this.handleIdentityCancel}
+                        onSave={this.handleIdentitySave}
+                      />
+                      <FavoriteDaosProvider>
+                        <ActivityProvider
+                          account={account}
+                          daoDomain={daoAddress.domain}
+                          web3={web3}
+                        >
+                          <PermissionsProvider
+                            wrapper={wrapper}
+                            apps={appsWithIdentifiers}
+                            permissions={permissions}
+                          >
+                            <div css="position: relative; z-index: 1">
+                              <Wrapper
+                                visible={mode === APP_MODE_ORG}
+                                account={account}
+                                apps={appsWithIdentifiers}
+                                appsStatus={appsStatus}
+                                canUpgradeOrg={canUpgradeOrg}
+                                connected={connected}
+                                daoAddress={daoAddress}
+                                daoStatus={daoStatus}
+                                historyBack={this.historyBack}
+                                historyPush={this.historyPush}
+                                locator={locator}
+                                onRequestAppsReload={
+                                  this.handleRequestAppsReload
+                                }
+                                onRequestEnable={enableWallet}
+                                permissionsLoading={permissionsLoading}
+                                repos={repos}
+                                signatureBag={signatureBag}
+                                transactionBag={transactionBag}
+                                walletNetwork={walletNetwork}
+                                walletProviderId={walletProviderId}
+                                walletWeb3={walletWeb3}
+                                web3={web3}
+                                wrapper={wrapper}
+                              />
+                            </div>
+                          </PermissionsProvider>
+
+                          <Onboarding
+                            selectorNetworks={selectorNetworks}
```

Comment:
I think we'll still need the balance information, to display the error on trying to create an org without enough ETH

Reply:
Done in https://github.com/aragon/aragon/pull/921/commits/1f97d45fa34d03733770cd47b5eb5f43f81e433a